### ISSUE TITLE
World Border disappears apon death or killing yourself.

### ISSUE THREAD

**Issue by johncsuti** [BentoBoxWorld/Border#22]
#### Describe the bug

Title says it all or watch the video for proof. I have no idea what is going on or why.

#### Steps to reproduce the behavior

1.) Kill yourself.
2.) Free roam the world.

#### Expected behavior

Persistent world border.

#### Screenshots and videos (Optional)

https://youtu.be/KqdaI0is9LM

##### BentoBox Version (Mandatory)

```
Running PAPER 1.15.2.
BentoBox version: 1.11.0
Database: JSON
Loaded Game Worlds:
bskyblock_world (BSkyBlock): Overworld, [nether_color]Nether, [end_color]End
Loaded Addons:
Biomes 1.6.0.1-SNAPSHOT-b149 (ENABLED)
Border 1.0.6 (ENABLED)
BSkyBlock 1.9.0 (ENABLED)
Challenges 0.8.0-1.6.0 (ENABLED)
ExtraMobs 1.10.0.1-SNAPSHOT-b0 (ENABLED)
Level 1.9.0 (ENABLED)
Limits 1.9.0 (ENABLED)
MagicCobblestoneGenerator 1.9.0-SNAPSHOT-b92 (ENABLED)
TwerkingForTrees 1.1.2 (ENABLED)
Warps 1.9.0 (ENABLED)
```

#### Plugins (Optional)

```
N/A
```
 
### Additional context (Optional)

N/A

**Comment by tastybento** [comment]
Please try the latest build on the CI server.

**Comment by johncsuti** [comment]
This build server?
https://travis-ci.com/BentoBoxWorld/Border

**Comment by johncsuti** [comment]
Never mind found it testing it now

**Comment by tastybento** [comment]
The latest should work. Let me know if still doesn’t.

**Comment by johncsuti** [comment]
I’ll give it a shot when I get home from work thanks for your help.

**Comment by johncsuti** [comment]
Fixed.

**Comment by johncsuti** [comment]
Re opened as it is now also happening on aoneblock

**Comment by tastybento** [comment]
Fixed in 1.0.7
